webapp/README.md

```diff
@@ -8,51 +8,76 @@ Webapp is a Go application that will serve up a static web portal to make it eas
 
 
 ## Packaged Setup/Run
-For running `webapp` in a package environment, like the default [SCIONLab](https://www.scionlab.org) environment, it will require command-line options for `webapp` to find the tools its requires. In most cases, the default `scionlab` and `scion-apps` packages will need to be specified on the command-line.
+For running `webapp` in a packaged environment, like the default [SCIONLab](https://www.scionlab.org) environment, it now uses command-line options for `webapp` to find the tools it requires.
 
-To run the Go Web UI at a specific address `-a` and port `-p` like 0.0.0.0:8000 for a SCIONLab VM use:
+To install from our packages, install `webapp` including its `scion-apps` dependencies:
 ```shell
-webapp \
+sudo apt install scion-apps-webapp
+```
+Alternatively, the following will install all `scion-apps` repo binaries:
+```shell
+sudo apt install scion-apps-*
+```
+
+Start the `webapp` service:
+```shell
+sudo systemctl start scion-webapp
+```
+
+Ensure the `webapp` service is running:
+```shell
+sudo systemctl status scion-webapp
+```
+
+Now, open a web browser at [http://127.0.0.1:8000](http://127.0.0.1:8000), to begin.
+
+Logs from `webapp` can be monitored:
+```shell
+journalctl -u scion-webapp -e
+```
+
+You won't need to add all the parameters yourself as the `scion-webapp.service` will do this for you, but as background information the service will run with the following command-line:
```

Comment:
Instead of copy-pasting all the stuff here, we could just point the user to `cat` the `scion-webapp.service` if he wants to see the defaults